packages/react-components/react-carousel-preview/src/components/CarouselNavButton/useCarouselNavButtonStyles.styles.ts

```diff
@@ -21,13 +21,13 @@ const useStyles = makeStyles({
     backgroundColor: tokens.colorBrandBackground,
   },
   rootUnselected: {
-    ...shorthands.border(0),
-    ...shorthands.borderRadius('50%'),
-    ...shorthands.padding('0px'),
-    ...shorthands.outline(tokens.strokeWidthThin, 'solid', 'transparent'), // For high contrast
+    border: 'none',
+    borderRadius: '50%',
+    padding: '0px',
+    outline: `${tokens.strokeWidthThin} solid transparent`, // For high contrast
     ...createCustomFocusIndicatorStyle({
-      ...shorthands.outline(tokens.strokeWidthThick, 'solid', tokens.colorStrokeFocus2),
-      ...shorthands.borderRadius(tokens.borderRadiusMedium),
+      outline: `${tokens.strokeWidthThick} solid ${tokens.colorStrokeFocus2}`,
+      borderRadius: tokens.borderRadiusMedium,
       ...shorthands.borderColor('transparent'),
```

Comment:
Seems like it missed borderColor shorthands

Reply:
Nvm, these are not supported yet.

Reply:
`borderColor`, `borderStyle`, `borderWidth` are still not supported, more details in  https://github.com/microsoft/griffel/issues/531